FinniversKit/Sources/Recycling/GridViews/NeighborhoodProfile/NeighborhoodProfileBannerView.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import UIKit
+
+protocol NeighborhoodProfileBannerViewDelegate: AnyObject {
+    func neighborhoodProfileBannerDidSelectButton(_ view: NeighborhoodProfileBannerView)
+}
+
+final class NeighborhoodProfileBannerView: UIView {
+//    static func height(forTitle title: String, width: CGFloat) -> CGFloat {
+//        return title.height(withConstrainedWidth: width, font: titleFont)
+//    }
+//
+//    private static let titleFont = UIFont.title3
+
+    // MARK: - Internal properties
+
+    weak var delegate: NeighborhoodProfileBannerViewDelegate?
+    
+    var text = "" {
+        didSet {
+            textLabel.setHTMLText(text)
+            textLabel.textAlignment = .center
+        }
+    }
+    
+    var buttonText = "" {
+        didSet { button.setTitle(buttonText, for: .normal) }
+    }
+    
+
+    // MARK: - Private properties
+    
+    private lazy var containerStackView: UIStackView = {
+        let stack = UIStackView(withAutoLayout: true)
+        stack.layer.borderWidth = 1
+        stack.layer.borderColor = .imageBorder
+        stack.layer.cornerRadius = .spacingS
+        stack.layer.masksToBounds = true
+        stack.axis = .vertical
+        stack.spacing = .spacingM
```

Comment:
Another shorthand for initializing `UIStackView`

```suggestion
        let stack = UIStackView(axis: .vertical, spacing: .spacingM, withAutoLayout: true)
        stack.layer.borderWidth = 1
        stack.layer.borderColor = .imageBorder
        stack.layer.cornerRadius = .spacingS
        stack.layer.masksToBounds = true
```